src/gpu/intel/ocl/micro_sdpa.hpp

```diff
@@ -100,14 +100,16 @@ struct micro_sdpa_t : public gpu_primitive_t {
                         "kq scales mask(%d) must equal kq zero point(%d) "
                         "mask",
                         kq_scales_mask, kq_zp_mask);
-            VDISPATCH_SDPA(utils::one_of(kq_scales_mask, 0, 1, 3, 11, 15),
-                    "unsupported mask for kq matmul(%d). must be 0, 1, 3, 11, "
-                    "or 15",
-                    kq_scales_mask);
-            VDISPATCH_SDPA(utils::one_of(kq_zp_mask, 0, 1, 3, 11, 15),
-                    "unsupported mask for kq matmul(%d). must be 0, 1, 3, 11, "
-                    "or 15",
-                    kq_zp_mask);
+            if (!desc()->kq_scales.has_default_values())
+                VDISPATCH_SDPA(utils::one_of(kq_scales_mask, 0, 1, 3, 11, 15),
```

Comment:
Main branch checks against 0, 1, 3, 7, 15 for vs_scales_mask and vs_zp_mask. What is going on with 7/11?